docs/nl/beginners/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Beginners
+meta:
+  - property: og:title
+    content: Beginners | Ethereum
+lang: nl-NL
```

Comment:
Same with this `lang` property, you can change to `nl` for consitency (you can likely find & replace all `nl-NL` with `nl`.